scripts/ansible/roles/substrate-systemd/templates/ee.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description={{ app_name }} Systemd Service 
+
+[Service]
+EnvironmentFile=/etc/default/{{ app_name }}
+ExecStart={{ app_path }} --chain alex --port 30363 --name $NAME --key $KEY --node-key $NODE_KEY --in-peers 100 --out-peers 50 -laura=debug,afg=debug,sync=trace,state-db=debug,gossip=debug
```

Comment:
maybe have the logging configuration also as a variable from the defaults file - maybe also the others like peer counts.

Reply:
Yip sounds good, was thinking about this before, especially if you want to make it configurable per host.

Reply:
Also removed --chain alex and --por 30363 (use binary defaults)